templates/selenium3.141.x-python37/src/__ALGO__.py

```diff
@@ -12,20 +13,26 @@ def get_chrome():
     return driver
 
 def get_phantomjs():
-    driver = webdriver.PhantomJS()
+    # PhantomJS must put logs in a place where it has write permissions
+    driver = webdriver.PhantomJS(service_log_path="/tmp/ghostdriver.log")
     return driver
 
 # API calls will begin at the apply() method, with the request body passed as 'input'
 # For more details, see algorithmia.com/developers/algorithm-development/languages
 def apply(input):
+    # note: you MUST have internet access enabled for this test work
     driver = get_chrome()
     #driver = get_phantomjs()
```

Comment:
Still not a fan of the commented out line. Why not have a `getDriver` method that takes in an enum that is either phantomjs or chrome?